docs/source/event-descriptors.rst

```diff
@@ -0,0 +1,210 @@
+Event Descriptors
+=================
+
+In the section on :doc:`documents`, we gave an overview of the four kinds of
+document. We presented an example Run Start, Event, and Run Stop, but we
+deferred detailed discussion of the Event Descriptor.
+
+Recall our example 'event' document.
+
+.. code-block:: python
+
+    # 'event' document (same as above, shown again for reference)
```

Comment:
"above" is out of date --- this is own its own page

Reply:
check